paddle/fluid/framework/tensor.cc

```diff
@@ -120,5 +120,18 @@ void Tensor::ResetHolderWithType(std::shared_ptr<memory::Allocation> holder,
   type_ = type;
 }
 
+int Tensor::dump(const char* name) {
```

Comment:
Can you make this method const? Otherwise I can't dump bias and weights in FC layer because they are const tensors.
```suggestion
int Tensor::dump(const char* name) const {
```